graphql-dgs-spring-webmvc-autoconfigure/src/test/kotlin/com/netflix/graphql/dgs/webmvc/autoconfigure/DgsWebMvcAutoConfigurationTest.kt

```diff
@@ -47,71 +47,71 @@ import org.springframework.test.web.servlet.setup.MockMvcBuilders
 
 class DgsWebMvcAutoConfigurationTest {
 
-    private val context = WebApplicationContextRunner().withConfiguration(
-        AutoConfigurations.of(
-            DgsWebMvcAutoConfiguration::class.java
-
-        )
-    )!!.withUserConfiguration(MockUserConfiguration::class.java)
+    private val context =
+        WebApplicationContextRunner()
+            .withConfiguration(AutoConfigurations.of(DgsWebMvcAutoConfiguration::class.java))
+            .withUserConfiguration(MockUserConfiguration::class.java)
 
     @Test
     fun objectMapperAvailable() {
         context.run { ctx ->
-            Assertions.assertThat(ctx).hasSingleBean(ObjectMapper::class.java)
-            Assertions.assertThat(ctx).getBeans(ObjectMapper::class.java).containsKey("dgsObjectMapper")
-            val modules = ctx.getBeansOfType(ObjectMapper::class.java)["dgsObjectMapper"]?.registeredModuleIds?.contains("com.fasterxml.jackson.datatype.jsr310.JavaTimeModule")
-            Assertions.assertThat(modules).isTrue()
+            assertThat(ctx).hasSingleBean(ObjectMapper::class.java)
+            assertThat(ctx).getBeans(ObjectMapper::class.java).containsKey("dgsObjectMapper")
+            // Expecting the JavaTimeModule from the dgsObjectMapper provided via MockUserConfiguration
+            val modules = ctx.getBeansOfType(ObjectMapper::class.java)["dgsObjectMapper"]
+                ?.registeredModuleIds?.contains(MockUserConfiguration.jacksonJavaTimeModule.moduleName)
```

Comment:
There is a lesson here, use the _Module's_ `moduleName` instead of the assumed string, e.g. `"com.fasterxml.jackson.datatype.jsr310.JavaTimeModule"`.